### PR TITLE
Added simple sprint functionality

### PR DESCRIPTION
**Pull request by Luckyleck**
When I opened your deployment and got to see my world in first person I was super impressed! Thank you for building this. While exploring, I thought this could definitely use a sprint feature. I added one. Used the shift key and added a multiplier to the movement logic. I did also run formatting, I hope that is not an issue, but nothing major was changed. 

image
image

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,142 +1,146 @@
...
-					<p>Drag and drop</br>Townscaper OBJ file here</p>
-				</div>
-				</br>
-				<span style="font-size:36px">Click to play</span>
-				<br /><br />
-				Move: WASD<br/>
-				Jump: SPACE<br/>
-				Look: MOUSE
+<head>
+	<title>Three Scaper</title>
```
Comment (meliharvey) [inline index.html]: Can you just change this title to `Threescaper`

**Review by meliharvey — CHANGES_REQUESTED** [review]
Hi Alex, 

Thanks for making these improvements. I tested it out and I like the sprint functionality. 

I just asked for one minor change to update the HTML title to Threescaper.

File: index.html (hunk 1 above)
Comment (Luckyleck) [reply index.html]: Hey Meli, just updated it. Hope I did it properly :)

**Review by meliharvey — APPROVED** [review]
Thanks!

**Pull request merged**